guides/common/modules/con_security-features.adoc

```diff
@@ -0,0 +1,52 @@
+[id="security-features"]
+= Security features in {Project}
+
+When deploying {Project}, consider implementing security features that enhance the overall security of your system.
+{Project} supports multiple security mechanisms to provide additional layers of protection, which you can enable during or after the installation of {Project}, if your use case requires them. 
```

Comment:
Should we mention that these security considerations are applicable to both ProjectServer and SmartProxyServers?

Reply:
@jherrman Actually, can you address this? I missed it during review.

Reply:
@Lennonka , already did: https://github.com/theforeman/foreman-documentation/pull/3799/files#diff-5a886518aa8c58822600953b3d423d55082b31748c85b9f4810099e61c3de04aR9

Is this what you had in mind?

Cheers,
J.

Reply:
Oh, yes, that. Thanks!